ci-scripts/tx-query-hw-test.sh

```diff
@@ -9,7 +9,7 @@ export RUST_LOG=debug
 export RUST_BACKTRACE=1
 export RUSTFLAGS=-Ctarget-feature=+aes,+sse2,+sse4.1,+ssse3
 
-ls /dev/sgx
+ls /dev/isgx
```

Comment:
I guess this should be configurable -- e.g. on Azure (where CI runs), the device is `/dev/sgx`

Reply:
Ok